### DISCUSSION TITLE
Chat room?

### DISCUSSION THREAD

**Discussion by n-raine** [slint-ui/slint#971] [General] [answered]
Is there a chat room for Slint? I want to get involved but I don't want to spam the discussions with my silly newbie questions while I'm learning my way around.

**Comment by jrmoulton** [comment, marked as answer]
The README links to this chat :)
https://chat.slint-ui.com/

**Reply by n-raine** [reply]
Whoops, I thought I looked there but I must not have scrolled down far enough. Thanks.

**Reply by jrmoulton** [reply]
Haha no problem

**Comment by calvin2021y** [comment]
can not get the verify email from the chat room

**Reply by tronical** [reply]
I looked into the logs of the mail server sending those out and I can see that the verification email was sent and accepted for delivery by the mail server responsible for handling your email. Is it possible that the mail was perhaps marked as spam?

**Reply by calvin2021y** [reply]
not likely. I use gmail and already check the trash and spam folder.

**Reply by tronical** [reply]
The email address registered with your Mattermost account is not a gmail address.
